src/jsifier.js

```diff
@@ -281,8 +287,10 @@ function(${args}) {
         error('DISABLE_EXCEPTION_THROWING was set (likely due to -fno-exceptions), which means no C++ exception throwing support code is linked in, but exception catching code appears. Either do not set DISABLE_EXCEPTION_THROWING (if you do want exception throwing) or compile all source files with -fno-except (so that no exceptions support code is required); also make sure DISABLE_EXCEPTION_CATCHING is set to the right value - if you want exceptions, it should be off, and vice versa.');
         return;
       }
-      const num = +symbol.split('_').slice(-1)[0];
-      addCxaCatch(num);
+      if (!(symbol in LibraryManager.library)) {
```

Comment:
Maybe add a comment as to what situation this handles? I'm not sure from reading the code.

Reply:
There is a fairly long comment just above, but I added another specific one here.